Description: fix libnames in pc files
 We cannot apply versioning as an after effect. The build produces a
 libweave.so.${abi}. There is no version as part of the basename.
 .
 libweave (0.0.1-1) unstable; urgency=low
 .
   * Initial release (Closes: #nnnn)
Author: Stefan Sauer <ensonic@google.com>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- a/libweave-test.pc.in
+++ b/libweave-test.pc.in
@@ -4,4 +4,4 @@
 Description: Weave test library
 Version: ${bslot}
 Requires: @PRIVATE_PC@
-Libs: -lweave-test-${bslot}
+Libs: -lweave-test
--- a/libweave.pc.in
+++ b/libweave.pc.in
@@ -4,4 +4,6 @@
 Description: Weave device library
 Version: ${bslot}
 Requires.private: @PRIVATE_PC@
-Libs: -lweave-${bslot}
+Libs: -lweave
+Cflags: -I/usr/include/weave
+
